deployments/kubernetes/Dockerfile.install-cni

```diff
@@ -1,17 +1,16 @@
-FROM amd64/alpine:3.8
+FROM ubuntu:xenial
 
 LABEL description="Istio CNI plugin installer."
 
-# add bash to allow install-cni.sh to trap process signals
-RUN apk --update add jq bash && \
-    rm -rf /var/lib/apt/lists/* && \
-    rm /var/cache/apk/*
+RUN apt-get update && apt-get install -qqy --no-install-recommends \
+jq='1.5*' \
+&& rm -rf /var/lib/apt/lists/*
```

Comment:
nit: could you reformat this block of code? I'd prefer each line to end in `&& \` like previously.

Reply:
@venilnoronha I've been following the same style as in https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run. With this style, adding or removing a package only changes a single line.